docs-chef-io/content/server/upgrades.md

```diff
@@ -54,12 +54,20 @@ Chef Infra Server 14.13 supports external OpenSearch for indexing. To migrate fr
 
 1. Set the `opensearch['external']` attribute to `true`.
 1. Set the `opensearch['external_url']` attribute to the external OpenSearch URL.
+1. Set the `opscode_erchef['search_queue_mode']` attribute to `batch`.
+1. Set the `opscode_erchef['search_provider']` attribute to `opensearch`.
+1. Set the `opscode_erchef['search_auth_username']` attribute to OpenSearch username.
+1. Set the `opscode_erchef['search_auth_password']` attribute to OpenSearch password.
 
 For example:
 
 ```bash
+opscode_erchef['search_queue_mode'] = 'batch'
+opscode_erchef['search_provider'] = 'opensearch'
 opensearch['external'] = true
 opensearch['external_url'] = "http://127.0.0.1:9200"
+opscode_erchef['search_auth_username'] = 'OPEN_SEARCH_USER'
```

Comment:
Des this need to be the admin user o in Opensearch?

Reply:
The user should be able to create index(chef) but not admin.

Reply:
It would be helpful to clarify that in the docs.